.bazelci/presubmit.yml

```diff
@@ -0,0 +1,11 @@
+---
+buildifier: latest
+platforms:
+  ubuntu1804:
+    working_directory: examples/basic
+    build_targets:
```

Comment:
i think `--fat_apk_cpu=arm64-v8a,x86` and `--android_crosstool_top=@androidndk//:toolchain` will also be needed, which i think can be added like this:
https://github.com/bazelbuild/bazel/blob/d3bef49f9182f6bcb6f609530970a7945ce51cbf/.bazelci/presubmit.yml#L117

(and for `macos` below)

Reply:
oops yep, sorry